src/AbstractCas.php

```diff
@@ -165,6 +161,10 @@ public function requestTicketValidation(
 
         $parameters += ['ticket' => $ticket];
 
+        if ('' === $parameters['ticket']) {
```

Comment:
Why this here?

Reply:
This acts as a replacement for the `supportAuthentication()` which was removed above. Saves a round trip to ECAS servers with an empty token.

Reply:
I would then prefer to do:

```php
    public function supportAuthentication(array $parameters = []): bool
    {
        return array_key_exists('ticket', $parameters) || Uri::hasParams($this->getServerRequest()->getUri(), 'ticket');
    }

```